iron-focusables-helper.html

```diff
@@ -127,18 +127,17 @@
         //  </div>
         // TODO(valdrin) support ShadowDOM v1 when upgrading to Polymer v2.0.
```

Comment:
Not really. So this helper does give you a sorted list of tabbable nodes; that example above this TODO explains what would happen with native shadow dom. I should definitely get back to this & fix it.